ui/components/multichain/pages/page/components/header/header.tsx

```diff
@@ -54,17 +54,20 @@ export const Header = ({
       endAccessory={endAccessory}
       {...props}
     >
-      <Text
-        display={Display.Block}
-        variant={TextVariant.bodyMdBold}
-        textAlign={TextAlign.Center}
-        paddingInlineStart={8}
-        paddingInlineEnd={8}
-        ellipsis
-        {...textProps}
-      >
-        {children}
-      </Text>
+      {children && (
+        <Text
+          as="h2"
+          display={Display.Block}
+          variant={TextVariant.headingSm}
+          textAlign={TextAlign.Center}
+          paddingInlineStart={8}
+          paddingInlineEnd={8}
+          ellipsis
+          {...textProps}
+        >
+          {children}
+        </Text>
+      )}
```

Comment:
Adding logic to only render the h2 tag if children exist see [ui/components/app/assets/nfts/nft-details/__snapshots__/nft-full-image.test.js.snap](https://github.com/MetaMask/metamask-extension/pull/29308/files#diff-c95cb922155464b2100e6eb0d924416a8e2fd591c005f509248a34370a52901dL20-L22)